simtools/db_handler.py

```diff
@@ -182,21 +180,32 @@ def getModelParameters(
 
     def exportFileDB(self, dbName, dest, fileName):
         """
-        Get a file from the DB and write it to disk.
+        Get file from the DB and write to disk.
 
         Parameters
         ----------
         dbName: str
             Name of the DB to search in.
         dest: str or Path
             Location where to write the file to.
-        fileName: str
+        file_name: str
             Name of the file to get.
+
+        Returns
+        -------
+        GridOut._id
+            :class:`~gridfs.grid_file.GridOut._id`
```

Comment:
I don't mind keeping the return ID, but maybe write this in a more friendly way? I.e., 
```
fileID: GridOut._id
    the database ID the file was assigned when it was inserted to the DB.
``` 
It's a bit clearer than GridOut info which someone has to look up.

Reply:
Hmm... I copied from the grid module exactly as they use there. shouldnt we be consistent?